packages/ui/src/working-groups/modals/ApplicationDetailsModal/GeneralDetails.tsx

```diff
@@ -0,0 +1,47 @@
+import BN from 'bn.js'
+import React from 'react'
+
+import { UnknownAccountInfo } from '../../../accounts/components/UnknownAccountInfo'
+import { AboutDateColumn } from '../../../common/components/AboutDateColumn'
+import { TokenValue } from '../../../common/components/typography'
+import { openingTitle } from '../../helpers'
+import { WorkingGroupApplication } from '../../types/WorkingGroupApplication'
+
+interface Props {
+  application: WorkingGroupApplication
+}
+
+export const GeneralDetails = React.memo(({ application }: Props) => (
+  <>
+    <table>
```

Comment:
We have already styled-components for these here: [https://github.com/Joystream/pioneer/blob/a708b68772d2b79306c4c94ec16fe9ca39971914/packages/ui/src/memberships/components/MemberProfile/MemberDetails.tsx#L42](https://github.com/Joystream/pioneer/blob/a708b68772d2b79306c4c94ec16fe9ca39971914/packages/ui/src/memberships/components/MemberProfile/MemberDetails.tsx#L42)

What's missing is extracting them so they become reusable, please do it in a follow-up:

*   extract `<AboutTable/>` and other base components to `common/components`
*   use them for General Details
*   make side pane modal general (I've seen some duplication there)

Reply:
As for duplication: `<SidePaneGlass onClick={onBackgroundClick}>` – this should be hidden so you wouldn't have to define `onBackgroundClick` every time.